### PR TITLE
Right click updates VC-1934

### PR DESCRIPTION
**Pull request by nikitahl**

### CODE REVIEW

File: public/components/panels/addContent/addContentPanel.js [hunk 1]
```diff
@@ -165,18 +165,18 @@ export default class AddContentPanel extends React.Component {
     let settingsControl
     if (
       (
-        roleManager.can('editor_content_element_add', roleManager.defaultAdmin()) &&
+        roleManager.can('editor_content_element_add', roleManager.defaultTrue()) &&
```
Comment (konutis) [inline public/components/panels/addContent/addContentPanel.js:168]: why did you change this?

**Comment by nikitahl** [comment]
Wrong commits
